Improve SEO metadata across all docs pages (#397)
powersync-ja/powersync-docs: client-sdks/orms/kotlin/room.mdx

---
title: "Room (Alpha)"

> description: "Use Android Room with PowerSync's Kotlin Multiplatform SDK for connection pooling, CRUD operations, and raw SQLite table access (alpha)."

Reviewer: ```suggestion
description: "Learn how to use Android Room with PowerSync's Kotlin SDK."
```
